osidb/models.py

```diff
@@ -2733,6 +2747,83 @@ def __str__(self):
         return self.username
 
 
+class FlawDraft(FlawMixin, AlertMixin):
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created_dt = models.DateTimeField(auto_now_add=True)
+    updated_dt = models.DateTimeField(auto_now=True)
```

Comment:
Because this model lives in OSIDB only and `TrackingMixin` does a lot of specific logic for models for which the tracking is done externally.

I could create an `InternalTrackingMixin` that is simpler (simply use `auto_now_add` and `auto_now`) or something but I don't know how useful it'll be at this time.

Reply:
It does all what is done here - introduces two `dt` attributes and implement `auto_now_add` and `auto_now` logic. Additionally it allows rewriting of this automation by explicit values which may or may not be useful but also mid-air collision detection. I see no reason why we should not care for the mid-air collisions in the case of drafts.